indico/modules/categories/templates/management/settings.html

```diff
@@ -0,0 +1,66 @@
+{% from 'forms/_form.html' import form_header, form_rows, form_fieldset, form_footer %}
+
+{% extends 'categories/management/base.html' %}
+
+{% block title %}
+    {% trans %}Settings{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {{ form_header(form) }}
+        {{ form_rows(form, fields=form.BASIC_FIELDS) }}
```

Comment:
Saw it in other places. It kind of makes sense to delimit the form and differentiate it from the content, IMHO.